.docker/Dockerfile.php

```diff
@@ -5,9 +5,7 @@
 
 FROM amazeeio/php:${PHP_IMAGE_VERSION}-fpm-${LAGOON_IMAGE_VERSION}
 
-RUN apk add --no-cache gmp gmp-dev \
-    && docker-php-ext-install gmp \
-    && docker-php-ext-configure gmp
+RUN apk add --update clamav clamav-libunrar \
```

Comment:
```suggestion
```

Reply:
This duplicates the follow-up command (`RUN apk add --no-cache --update clamav clamav-libunrar`) below